qutebrowser/keyinput/basekeyparser.py

```diff
@@ -189,6 +191,7 @@ def __repr__(self) -> str:
                               passthrough=self.passthrough,
                               supports_count=self._supports_count)
 
+    # pylint: disable=redefined-outer-name
```

Comment:
Please rename the `message` argument instead (to e.g. `msg` or `text`).

Reply:
Thanks, I didn't really like disabling that for the whole function, but kind of relied on your feedback here.